src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -261,9 +261,9 @@
                     };
 
                     if (json.missing.length || json.updated.length)
-                        $('#changedFiles').html('<div class="alert alert-warning">'+ translations.changesDetected +'</div>');
+                        $('#changedFiles').html('<div class="alert alert-warning"><ul><li>'+ translations.changesDetected +'</li></ul></div>');
                     else
-                        $('#changedFiles').html('<div class="alert alert-success">'+ translations.noChangeDetected +'</div>');
+                        $('#changedFiles').html('<div class="alert alert-success"><ul><li>'+ translations.noChangeDetected +'</li></ul></div>');
```

Comment:
You just display a string, why don't you use `<p></p>` instead?

Reply:
This is how you should render a notification block, according to UI kit, the "compromised" page have not been designed :+1: